dashboards-observability/public/components/visualizations/charts/bar/bar.tsx

```diff
@@ -33,6 +29,18 @@ export const Bar = ({ visualizations, layout, config }: any) => {
     },
     vis: visMetaData,
   }: IVisualizationContainerProps = visualizations;
+
```

Comment:
You could destruct visMetaData at line 30 directly without renaming. Please apply this code review to all other charts that have this similar problem.

Reply:
Hi @mengweieric, this is already taken care, can you please check once https://github.com/opensearch-project/observability/pull/1106/files